stale.yml

```diff
@@ -0,0 +1,40 @@
+# Configuration for probot-stale - https://github.com/probot/stale
```

Comment:
this is all well and good but it ain't working is it

Reply:
Hmm, would the bot pick this up?

Reply:
I will remove this and restore the contributing text.

Reply:
I have reinstalled the app, hopefully it starts working now

Reply:
It would be great if the bot did pick this up

Reply:
I wonder how we can test it.